src/backproject.jl

```diff
@@ -139,17 +139,17 @@ function backproject!(
     image .= zero(plan.T) # must be initialized as zero
     if plan.mode === :fast
         [plan.add_img[i] .= zero(plan.T) for i = 1:plan.nthread]
-        Threads.@threads for i in index
+        Threads.@threads for (i, viewidx) in collect(enumerate(index))
```

Comment:
```suggestion
        Threads.@threads for (i, viewidx) in enumerate(index)
```
collect is not needed

Reply:
`@threads` does not work with generic iterative variables like`enumerate`. See [this](https://discourse.julialang.org/t/enumerate-not-suported-for-threads/37670).

Reply:
Oh!  OK.  I will retract the suggestion.